frontend/src/components/project/EditProject.tsx

```diff
@@ -40,11 +40,15 @@ export const EditProject = () => {
   const client = useSelector((state: ConfacState) => state.clients.find(x => x._id === project.client.clientId) || getNewClient());
   const hasProjectMonths = useSelector((state: ConfacState) => state.projectsMonth.some(pm => pm.projectId === params.id));
   const [needsSync, setNeedsSync] = useState<{consultant: boolean, client: boolean}>({consultant: false, client: false});
+  const [copyId, setCopyId] = useState(0);
+  const useCopy = (id: number) => {
```

Comment:
Let op!!

De `use` prefix wordt gebruikt om aan te geven dat het een React Hook is
Maar in dit geval is het niet echt een hook, het is gewoon een functie
Hooks worden buiten een component gedefinieerd en worden dan binnen component(s) gebruikt

Reply:
Oké, merci voor de correctie.